opentech/apply/projects/views.py

```diff
@@ -54,32 +55,35 @@ def get_context_data(self, **kwargs):
         context['contracts'] = contracts
         return context
 
-    def get_contracts(self):
+    def get_contracts(self, project):
         """
         Get approved AND signed contracts for the current project
 
-        When an unsigned and unapproved contract is newer than the signed and
-        approved ones we add that to the top of the list.
+        When an unsigned OR (signed AND unapproved) contract is newer than the
+        signed and approved ones we add that to the top of the list.
         """
-        project = self.get_object()
-
         contracts = (project.contracts.select_related('approver')
                                       .order_by('-created_at'))
 
         if not contracts.exists():
             return
 
         latest_contract = contracts.first()
-        if not (latest_contract.is_signed and latest_contract.approver):
-            # remove unsigned contracts EXCEPT this PK
-            exclude_pks = (contracts.filter(is_signed=False)
-                                    .exclude(pk=latest_contract.pk)
-                                    .values_list('pk', flat=True))
-            contracts = contracts.exclude(pk__in=exclude_pks)
-        else:
-            contracts = contracts.exclude(is_signed=False)
-
-        return contracts
+
+        if latest_contract.is_signed and latest_contract.approver:
+            return contracts.exclude(is_signed=False)
```

Comment:
As the docstring says this is "signed **and approved**" 

```python
return contracts.filter(is_signed=True, approver__isnull=False)
```

Reply:
Fixed in dafa21d8